src/brevitas/core/scaling/runtime.py

```diff
@@ -81,21 +75,12 @@ def __init__(
             self,
             restrict_scaling_impl: Module,
             restrict_threshold_impl: Module,
-            scaling_shape: Tuple[int, ...],
-            scaling_min_val: Optional[float],
-            affine_rescaling: bool,
-            affine_shift_scale: bool,
-            dtype: Optional[torch.dtype],
-            device: Optional[torch.device]) -> None:
+            affine_rescaling_impl: Optional[Module] = None,
```

Comment:
I think this looks like makes things cleaner, but actually it will only complicate stuff at dependency injection time